### PR TITLE
add sources, triggers, and attrib logic

### PR DESCRIPTION
**Pull request by nancymic2**
Fixes
- b/275551572
- b/277375658
- b/273505307

Changes proposed in this pull request:
- add sources, triggers, and attribution logic docs from ARA handbook
- https://pr-6690-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/attribution-reporting/register-attribution-source/
- https://pr-6690-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/attribution-reporting/register-attribution-trigger/
- https://pr-6690-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/attribution-reporting/change-attribution-logic/

### CODE REVIEW

**Comment by nancymic2** [comment]
Approved by Maud

File: site/en/docs/privacy-sandbox/attribution-reporting/register-attribution-source/index.md [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+that is separate from the navigation, and that you can completely control. By
+specifying an explicit value for `attributionsrc`, you're instructing the
+browser to make that extra request and configuring its destination.
+
+Click the tab above for <a href="#tabs">Views step 1</a>
+
+  </web-tab>
+  <web-tab title="Views step 1">
+<p>
+
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/attribution-reporting/register-attribution-source/index.md:206]: We should punctuate this sentence on both tabs, however, my issue with this instruction is it's a bit confusing. We're redirecting them to the tab with the link so what we need to say is something like "For instructions on how to register..., go to [the tab](# link back to the tabs) and select **Views step 1**."

**Review by anusmitaray — APPROVED** [review]

File: site/en/docs/privacy-sandbox/attribution-reporting/register-attribution-trigger/index.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+```
+
+{% Aside 'important' %}
+*   The origin for src must match the origin that performed source registration. 
+*   An attribution can only be triggered on a page whose [eTLD+1](https://web.dev/same-site-same-origin/#%22schemeful-same-site%22) matches the site that was provided in destination upon source registration. 
+{% endAside %}
+
+#### Using a script tag
+
+You can perform trigger registration with a script tag; it behaves identically to &lt;img>. The following code samples illustrate the use of `fetch()` and `XMLHttpRequest()` (XHR).
```
Comment (rachelandrew) [inline site/en/docs/privacy-sandbox/attribution-reporting/register-attribution-trigger/index.md:145]:
```suggestion
You can perform trigger registration with a script tag; it behaves identically to `<img>`. The following code samples illustrate the use of `fetch()` and `XMLHttpRequest()` (XHR).
```

**Review by rachelandrew — APPROVED** [review]
Just a couple of minor nits and I'm in agreement with the comment Anusmita made about the tabs.

File: site/en/docs/privacy-sandbox/attribution-reporting/register-attribution-source/index.md (hunk 1 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/attribution-reporting/register-attribution-source/index.md:206]: good idea thanks!

File: site/en/docs/privacy-sandbox/attribution-reporting/register-attribution-source/index.md [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+window.open('...', '_blank', attributionsrc=${encodedUrl1}
+  attributionsrc=${encodedUrl2})
+```
+
+In such cases, both URLs receive `navigation-source-eligible` `attributionsrc`
+requests (requests that include the `Attribution-Reporting-Eligible` header).
+
+#### `attributionsrc` with or without a value
+
+As you saw above, you can specify `attributionsrc` without a URL. You can also specify a
```
[the remark this answers is not in the document]
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/attribution-reporting/register-attribution-source/index.md]: fixed

**Pull request merged**